### PR TITLE
Desouls the Roboticist Jumpsuit, and adds new Roboticist Boilersuit

### PR DESCRIPTION
**Pull request by Moth-lantern**
## About The Pull Request
So with the lovely work of my partner Alchemyandmagic, I present new bespoke and soulless roboticist jumpsuit and skirt resprites, along with a new more mechanic-esque roboticist boilersuit as an alternate costume for the job 
Roboticist_Jfront
Roboticist_jskf
boilersuit_front
Boilerskirt_front
Roboticist_inhands
## Why It's Good For The Game
Visual upgrades are generally good and the roboticist jumpsuit was poorly shaded and out of date with the current direction the artstyle of ss13 is heading, as for the new outfit, roboticist and research in general is one of the jobs with very very few alternate costumes for player customization, and some may wish for their roboticists to have a more mechanic-like aesthetic 
## Changelog
:cl:
imageadd: Nanotransen has made uniform changes to the robotics lab, modernizing their wardrobe.
/:cl:

### CODE REVIEW

**Comment by Fikou** [comment]
> roboticist and research in general is one of the jobs with very very few alternate costumes for player customization

the 4 unique jumpsuits roboticists can get:

**Comment by dragomagol** [comment]
The boilersuit outfits scream "security" to me more than "robotics"

**Comment by AtomTheProphet** [comment]
They do look a bit like security, but they look absolutely wonderful!

**Comment by san7890** [comment]
image

what are these supposed to be, i don't really understand

**Comment by Moth-lantern** [comment]
> image
> 
> what are these supposed to be, i don't really understand

Camera that I used to take pics of the ingame sprites with, sorry lol

**Comment by san7890** [comment]
lol just hit `f5` to adminghost and then use your screenshot tool

**Comment by Moth-lantern** [comment]
> lol just hit `f5` to adminghost and then use your screenshot tool

...oh.

**Comment by san7890** [comment]
it's okay not many people are aware of the number of debug tools we have and how they can be leveraged

**Comment by EricZilla** [comment]
Never liked the roboticists suit, but I like these much more, big fan of the new one.

(You can also windows + shift + S to take a screengrab)

**Comment by Moth-lantern** [comment]
oh yeah all these are valid options they just don't give nice zoomed in screenshots

**Comment by VladinXXV** [comment]
> oh yeah all these are valid options they just don't give nice zoomed in screenshots

I take these kinda screenshots all the time. Here's my advice:

Preferences > Open Game Preferences > Gameplay. Disable ambient occlusion.
image
_Note: Ambient occlusion doesn't seem to update automatically. After you disable it, try aghosting then re-entering your body._

Move to the "UI" section of the preferences now. In the "Pixel Scaling" dropdown, select "Pixel Perfect 3x". If this is too zoomed in, you can also try using 2x, but I wouldn't recommend lower than that.
image

The "photobooth" holodeck has a completely textureless white floor. This is my personal favorite, but on a case-by-case basis you may prefer using a more natural setting on the station.

Admin ghost (`F5` should work, but there's also a verb for it in the Admin tab). You'll probably want fullbright, but this can vary too. You can do this by clicking "Toggle Darkness" in the Ghost tab until you don't see any more shadows.
image

For actually taking the picture, I just use `Win + Shift + S`. You can paste the result directly to your PR from the clipboard using `Ctrl + V`.

Additionally tips:
For mob sprites (item inhands, clothing, etc), I like to show them all off in one picture. While a ghost, if you Ctrl + Shift + LMB yourself, you'll spawn in as your character, naked. You can use this to make quick clones of yourself. If you don't care about having pretty test dummies, you can also spawn `/obj/item/debug/human_spawner`, which will spawn a random naked human wherever you click.

You can see an example of this combined with the rest of the tips in #70784. Or any of my PRs with screenshots, really.

File: code/modules/clothing/under/jobs/rnd.dm [hunk 1]
```diff
@@ -103,6 +103,23 @@
...
+/obj/item/clothing/under/rank/rnd/roboticist/boiler
+	desc = "It's a comfortable black boilersuit, perfect for hiding oil stains from your RD."
+	name = "roboticist's boilersuit"
+	icon_state = "robotics_boilersuit"
+	inhand_icon_state = null
+	resistance_flags = NONE
+
+/obj/item/clothing/under/rank/rnd/roboticist/boiler/skirt
+	name = "roboticist's boilerskirt"
+	desc = "It's a comfortable black boilersuit, perfect for hiding oil stains from your RD."
```
Comment (JohnFulpWillard) [inline code/modules/clothing/under/jobs/rnd.dm:115]:
```suggestion
	desc = "It's a comfortable black boilerskirt, perfect for hiding oil stains from your RD."
```

**Comment by JohnFulpWillard** [comment]
Roboticists already get a ton more alternative costumes than Scientist/Geneticist, do we really want to be adding more?

**Comment by Moth-lantern** [comment]
> Roboticists already get a ton more alternative costumes than Scientist/Geneticist, do we really want to be adding more?

Roboticist has four costumes if you for some reason count the skirt and suit as completely different clothing items. A lot of security jobs have up to 6 or 7 alternate costume sets. If job disparity in clothing mattered security would have half as many outfits. But as it stands I think its fine if roboticist gets one more set. Also to be clear scientist can wear these just as well as roboticist. its not particularly hard for a scientist to get into robotics.

**Review by Imaginos16 — APPROVED** [review]
Its… Okay. Certainly better than the old one though.

**Comment by Jacquerel** [comment]
image
Unless you are both talking about something I am missing this really doesn't look unreasonably bloated to me, so unless someone actually wants to block adding more outfits on those grounds I will merge it
@Fikou @JohnFulpWillard 

Also I like the practical boiler suit more than the anime plug suit to be honest

**Comment by Moth-lantern** [comment]
Yea originally the scope of this was going to be changing the default to the boilersuit, but there was a noticeable contingent of players who still liked the plugsuit at the downstream I was originally pring for.

**Review by Twaticus — CHANGES_REQUESTED** [review]
sorry i'd rather not have an alt robo outfit at this time
the resprites fine just not the boilersuit

**Comment by Moth-lantern** [comment]
> sorry i'd rather not have an alt robo outfit at this time the resprites fine just not the boilersuit

Could I ask for clarification on this comment? I would like to know specifically why it is a point of concern/contention for the future.

**Comment by san7890** [comment]
fwiw i really enjoy the thought of a boilersuit but i think maybe with some stylistic alterations it could definitely fill some other niche in the game (aside from being yet another alt outfit for a job)

**Comment by Moth-lantern** [comment]
I am not going to pr another sprite for a different job to wait two weeks and then get told no without context, I'm sorry. but thank you for merging the jumpsuit updates.

**Comment by san7890** [comment]
> I am not going to pr another sprite for a different job to wait two weeks and then get told no without context, I'm sorry. but thank you for merging the jumpsuit updates.

I don't see why no one quoted this but it's from https://hackmd.io/@tgstation/HyOAnhDPI#Alternate-job-uniforms

> Alternate job uniforms
Jobs do not need multiple alternate uniforms in their closets, If you want to add a alternate uniform, you can put it in a cargo crate, or **put it in costume vendors that are public access**.

**Pull request merged**